GoogleCloudPlatform/compute-image-tools
# VM image importer/wrapper

This implements the first version of VM image importer:
- does everything Daisy does with more user friendly parameters (similar to gcloud)
- automatically labels temporary and permanent resources (to support Image import UI)

Tested:
- unit tests
- end to end manual testing 

TODO: 
- Verify automatic Docker image is being built properly
- Add release image once latest image is verified
- Integration tests
